filament/backend/src/opengl/platforms/PlatformEGL.cpp

```diff
@@ -91,7 +91,11 @@ Driver* PlatformEGL::createDriver(void* sharedContext, const Platform::DriverCon
         return nullptr;
     }
 
+#if defined(__ANDROID__) || defined(FILAMENT_USE_EXTERNAL_GLES3) || defined(__EMSCRIPTEN__)
+    // PlatofrmEGL is used with and without GLES, but this function is only
+    // defined for the platforms above in gl_headers.h.
```

Comment:
Or slightly more accurately "but this function is only meaningful when GLES is used".

Reply:
Done.